code/modules/surgery/organs/eyes.dm

```diff
@@ -1,3 +1,5 @@
+#define NIGHT_VISION_DARKSIGHT_RANGE 2
```

Comment:
This is the damn same value of standard eyes' `see_in_dark`, kevinz.

Reply:
bruh what